modules/config_import.py

```diff
@@ -1,13 +1,54 @@
-def get_config():
+def get_language(language, platform):
     #Imports
     import os
     from configparser import ConfigParser
     from configparser import ExtendedInterpolation
     import json
+    print(os.getcwd())
+    #Get Config
+    conf = {}
+    os.chdir('config/languages')
+    config = ConfigParser(interpolation=ExtendedInterpolation())
+    config.read(language + '_return.ini')
+    if platform == 'windows': #Windows
+        conf['err_0'] = config['Return_Dos']['err_0']
```

Comment:
Did you think about use a for loop to do this assignments?
```suggestion
        dos_return = 'Return_Dos'
        for key in conf.keys():
            conf[key] = config[dos_return][key]
```